libs/core/langchain_core/tools/__init__.py

```diff
@@ -41,6 +41,7 @@
 from langchain_core.tools.base import (
     create_schema_from_function as create_schema_from_function,
 )
+from langchain_core.tools.convert import MethodTool as MethodTool
```

Comment:
```suggestion
from langchain_core.tools.convert import MethodTool
```

Reply:
Linter (ruff) requires this redundant alias